src/server/pps/server/api_server.go

```diff
@@ -477,7 +477,7 @@ func (a *apiServer) InspectJob(ctx context.Context, request *pps.InspectJobReque
 // listJob is the internal implementation of ListJob shared between ListJob and
 // ListJobStream. When ListJob is removed, this should be inlined into
 // ListJobStream.
-func (a *apiServer) listJob(ctx context.Context, pipeline *pps.Pipeline, outputCommit *pfs.Commit) ([]*pps.JobInfo, error) {
+func (a *apiServer) listJob(ctx context.Context, pipeline *pps.Pipeline, outputCommit *pfs.Commit, inputCommits []*pfs.Commit) ([]*pps.JobInfo, error) {
```

Comment:
Do we need to call InspectCommit on these? I.e. if someone `pc list-job -p mypipeline -i master` should we try to convert `master` to an ID?

Reply:
That is a good point, we should probably do the same for output commits as well.